py3status/modules/taskwarrior.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
```

Comment:
overall note, if you can, please make sure your code passes the `yapf` formatting tool